dev/tests/unit/Util/MagentoTestCase.php

```diff
@@ -3,35 +3,36 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
 
 namespace tests\unit\Util;
 
-use AspectMock\Test as AspectMock;
 use PHPUnit\Framework\TestCase;
 
 /**
  * Class MagentoTestCase
  */
 class MagentoTestCase extends TestCase
 {
+    /**
+     * @inheritDoc
+     */
     public static function setUpBeforeClass(): void
     {
-        if (!self::fileExists(DOCS_OUTPUT_DIR)) {
+        if (!file_exists(DOCS_OUTPUT_DIR)) {
```

Comment:
Please revert this change back 
\PHPUnit\Framework\Assert::fileExists will exist in version which support PHP 8